pl.js

```diff
@@ -0,0 +1,778 @@
+stats.fistkills Zabójstwa w wyniku pobiciu
+stats.thrownkills Zabójstwa w wyniku rzutu bronią białą
+stats.headshots Trafienia w głowę
+stats.wallbangs Trafienia przez ściany
+stats.sprays Użycia sprejów
+windows.host.default.settings Włączenie domyślnych ustwaień pozwala na przechowywanie postępu graczy oraz najwyższych wyników
+windows.host.default.commands *Komendy hosta są wyłączone w tym trybie: /noclip /god /smite itp
+windows.host.rewards Włączenie nagród KR zmusza serwer do używania domyślnych ustawień i pozwala graczom zdobywać KR z funduszy mapy:
+windows.store.wheels.sub Wypróbuj swoje swoje szczęście i zakręć kołem, aby móc wygrać nowe skiny!
+windows.store.currency * Ceny w USD mogą być przeilczone na Twoją lokalną walutę po wartości wymiany ustalonej przez dostawcę płatności lub bank
+windows.store.creator Pomyśl nad wsparciem swoich ulubionych twórców treści gry (CC) używając ich kodu
+windows.chall.customs Nie można zdobywać wyzwań w niestandardowych meczach
+twitch.link Czy jesteś pewien, że chcesz połączyć konto gry z kontem na Twitchu?
+twitch.unlink Czy jesteś pewien, że chcesz rozłączyć konto gry z kontem na Twitchu?
+windows.clan.leave Czy jesteś pewien, że chcesz opuścić ten klan?
+windows.clan.delete Czy jesteś pewien, że chcesz usunąć ten klan?
+windows.clan.kick Czy jesteś pewien, że chcesz wyrzucić {0} z klanu?
+windows.store.premium.purchase Czy jesteś pewien, że chcesz kupić {0} dni Premium?
+windows.playerlist.ban Czy jesteś pewien, że chcesz wyrzucić {0} z tej gry?
+windows.playerlist.flag Czy jesteś pewien, że chcesz oflagować {0}?
+windows.playerlist.unflag Czy jesteś pewien, że chcesz odflagować {0}?
+windows.playerlist.punish Czy jesteś pewien, że chesz podjąć akcję dotyczącą {0}?
+item.view.open Otwórz przeglądarkę modeli
+item.view.more Więcej informacji
+item.sell.cant Przedmiot nie może zostać sprzedany.
+item.sell.quick Czy na pewno chcesz szybko szprzedać {0}?
+social.profile.stats Statystyki ogólne
+social.profile.class Statystyki klasy
+social.profile.sales Sprzedaż
+social.profile.sales.market Sprzedaż rynkowa
+social.profile.sales Sprzedaż
+social.profile.skins Skiny [{0}]
+social.profile.skins.none Ten gracz nie opublikował żadnych skinów
+social.profile.games Gry [{0}]
+social.profile.games.none Ten gracz nie opublikował żadnych gier
+social.profile.assets Zasoby [{0}]
+social.profile.assets.none Ten gracz nie opublikował żadnych zasobów
+social.profile.mods Modyfikacje [{0}]
+social.profile.mods.none Ten gracz nie opublikował żadnych modyfikacji
+social.search Wyniki wyszukiwania
+social.market.stats.rare Najrzadzsze przedmioty
+social.market.stats.circulation W obiegu
+social.market.stats.units Liczba na całym świecie
+social.trades.in Przychodzące
+social.trades.in.head Przychodzące oferty ({0})
+social.trades.out Wychodzące
+social.trades.out.head Wychodzące oferty ({0})
+social.trades.history Historia
+social.trades.history.head Historia handlu ({0})
+social.trades.decline Odmów wszystko
```

Comment:
I think "Odrzuć wszystkie oferty" or just "Odrzuć wszystkie" (if you want the shorter version) sounds a bit better